### ISSUE TITLE
JScomments Not Working with Grav Beta

### ISSUE THREAD

**Issue by ghost** [Sommerregen/grav-plugin-jscomments#1]
FYI ... You may already be aware, JScomments does not show in the page options tab under GRAV Beta.

**Comment by Sommerregen** [comment]
Hi @umbergerba ,

yes I'm already aware of. Can you change this line (https://github.com/Sommerregen/grav-plugin-jscomments/blob/master/jscomments.php#L96) to

``` php
if (false === $inEvent and $blueprint->get('form/fields/tabs', null, '/')) {
 ...
}
```

Further can you check, which version of **JSComments** you have? I've only got _v1.2.10_, which I'm not sure it was the "latest" and last release of this plugin. If you have a higher version can you send me a download link to it, so that I can incorporate the changes? And does the rest of the plugin works fine with the beta?

**Comment by ghost** [comment]
Yes, that seems to have fixed the issue I mentioned. I will test the plugin some more now and let you know if I see any other issues; so far everything else seems fine.

You have the latest version that was released (1.2.10).

Thanks!
Brian U.

**Comment by Sommerregen** [comment]
Hi @umbergerba ,

**JSComments** now includes this fix. It is in the testing release. Thus, please make sure to switch to the "testing" channel in order to get the latest additions.

Thanks!
